### ISSUE TITLE
Liste taxons bloquée sur critères 1ere recherche

### ISSUE THREAD

**Issue by aferchal** [PnX-SI/gn_mobile_occtax#90]
En version 1.2.4 si je mets des critères de recherche sur la liste de taxon avec le début du nom de l'espèce, ça fonctionne, je trouve bien les taxons correspondants. Par contre si je veux ajouter un taxon au meme relevé la liste des taxons reste filtrée sur les 1ers critères et je n'ai pas vu comment faire un raz pour avoir la liste complète. Si je veux choisir un autre taxon il faut alors saisir les 1eres lettres du taxon pour que la liste se mette à jour. Bug ou fonctionnement normal ?

**Comment by sgrimault** [comment]
Bonjour,

Effectivement, un petit bug sur la recherche. Elle reste active entre deux ajouts de taxons. Seuls les filtres peuvent être éventuellement conservés.

**Comment by camillemonchicourt** [comment]
Le soucis semble se produire aussi quand on filtre en texte les taxons, qu'on en choisit un, qu'on passe à l'étape suivante, puis qu'on revient en arrière sure le choix des taxons.
Là le filtre texte reste actif et on ne peut pas l'enlever, à moins de saisir un autre recherche textuel.
On ne peut pas revenir à la liste de tous les taxons.

**Comment by AudreyEnGuyane** [comment]
Idem chez nous sous Android 11 x OccTax Mobile 2.3.0...
Seul moyen trouvé: sortir de la saisie puis retourner dans le relevé en cours... Pas top...

**Comment by DonovanMaillard** [comment]
Je ne sais pas si c'est le cas pour tout le monde,ni depuis quelle version. Mais je ne reproduis pas ce soucis sur la version 2.5.0.

**Comment by gildeluermoz** [comment]
Sur la 2.4.0 c'était tjs présent. Pas testé la 2.5.0.
Est-ce que Sébastien a travaillé à la correction de ce bug ?

**Comment by sgrimault** [comment]
@gildeluermoz 
Non, aucune correction n'a été faite pour l'instant. À prioriser avec les autres tâches :)
